ruby/vampires.rb

```diff
@@ -0,0 +1,56 @@
+puts "How many employees you will be processed?"
+num_empl = gets.chomp.to_i 
+
+
+i=0
+until i == num_empl
+
+	puts "What is your name?"
+	name = gets.chomp
+	name == "Drake Cula" || name == "Tu Fang"? name = true : name = false
+	
+	puts "What is your age? What year were you born?"
+	age = gets.chomp.split(' ').map(&:to_i)
+	0 < age[0] && age[0] < 100? age[0] = true : age[0] =false
+	(Time.new.year - 100) < age[1] && age[1] < Time.new.year ? age[1] =true : age[1] =false
+	age = age[0] && age[1]
+	puts age
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)" 
+	garlic = gets.chomp
+	garlic == "y"? garlic =true : garlic =false
+	
+	puts "Would you like to enroll in the company’s health insurance?(y/n)"
+	h_insur = gets.chomp
+	h_insur == "y"? h_insur =true : h_insur =false
+
+	x = ""
```

Comment:
x is a generic variable name, try and make this more focused to what it represents.